src/Screen/Fields/Relation.php

```diff
@@ -192,6 +194,18 @@ public function applyScope(string $scope, ...$parameters): self
         return $this;
     }
 
+    /**
+     * @param array $columns
+     *
+     * @return $this
+     */
+    public function searchIn(array $columns): self
```

Comment:
It may not always be required to specify a lot of fields,
 let's change it to `string|array` and wrap the parameter in `Arr::wrap`?

Reply:
Maybe?
```php
public function searchIn(...$columns): self

Reply:
Yes, that's okay too